cdc_fuse_fs/mock_config_stream_client.cc

```diff
@@ -0,0 +1,24 @@
+#include "dev/file_transfer/cdc_fuse_fs/mock_config_stream_client.h"
+
+namespace cdc_ft {
+
+absl::Status MockConfigStreamClient::StartListeningToManifestUpdates(
```

Comment:
Nit: Could probably just have used a gmock class for this.

Reply:
Maybe.